app/posts/register-trainee-teachers/2021-04-08-school-direct-routes.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Adding school direct routes
+description: Expanding the routes we support
+date: 2021-04-08
+---
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{% from "figure/macro.njk" import appFigure with context %}
+
+We’re continuing to expand our manual data entry for new training routes, now adding [School direct routes](https://beta-getintoteaching.education.gov.uk/ways-to-train).
+
+School direct represents two similar routes:
+
+* School direct (salaried)
+* School direct (fee funded)
+
+The school direct routes are similar to our previous routes, especially Provider led (postgrad). The main new addition is asking about schools - their lead school for fee funded courses, and their lead and employing schools for salaried course.
+
+For the moment we’ve included these new data fields as part of the task list section that collects the trainee’s start date and ID. This is broadly a collection of training details specific to the trainee, whereas  the other section ‘course details’ is generic details about the course they’re on. 
```

Comment:
I see a rouge extra space in this paragraph.

Reply:
Good catch! 👍